app.js

```diff
@@ -254,6 +253,7 @@
   }
 
   function calcAQI(Cp, Ih, Il, BPh, BPl) {
+    // The AQI equation https://forum.airnowtech.org/t/the-aqi-equation/169
```

Comment:
Neat, I just got this from [PurpleAir's API Google Doc](https://docs.google.com/document/d/15ijz94dXJ-YAZLi9iZ_RaBwrZ4KtYeCy08goGBwnbCU/edit) but that lacks any explanation, which this provides. As @obra something better would be great, but this is better than what was there!